packages/extension-store-canner/src/lib/canner/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { ObjectBasicInfo } from '@canner/canner-storage';
+import { chain } from 'lodash';
+
+// The pattern for listing vulcansql folder path
+export const vulcanFolderPathPattern = new RegExp('([a-zA-Z0-9-]+)/vulcansql');
+// The pattern for finding indicator.json file from vulcansql folder
+export const indicatorPathPattern = new RegExp(
+  `${vulcanFolderPathPattern.source.replace('\\', '')}/indicator.json`
```

Comment:
Why should we remove the backword slash, and is there only one backword slash in the file path?

Reply:
No, we couldn't, because the `vulcanFolderPathPattern.source` will transfer to string, and the `\` prefix in fronted of `/`, so we should remove it to blank:

![截圖 2023-06-06 下午6 05 36](https://github.com/Canner/vulcan-sql/assets/5389253/dadaeb3c-6914-436e-a788-d6f06792b4a4)